apps/jobs/constants/values.py

```diff
@@ -11,6 +11,15 @@
 
 GENDER = (("male", "Male"), ("female", "Female"), ("other", "Other"))
 
+HIRING_STATUS = (("hiring", "HIRING"), ("open to work", "OPEN TO WORK"))
+
+JOB_TYPE = (
```

Comment:
@Himan10 lets not store upper case and smaller case. lets keep it constant. either lower or upper.

Reply:
you can do the check instead of keeping in variable